SpywareFilter/sections/specific.txt

```diff
@@ -22,6 +22,7 @@
 ||nd.demdex.net/event?d_mid=$empty,important,domain=9now.com.au
 ||adc.nine.com.au/?token=$empty,important,domain=9now.com.au
 !
+||passnavi.evidus.com/js/*_tracking
```

Comment:
Is everything works fine with this rule?

Reply:
Also missed tracker:
`||track.edu-info.jp^`

 And broken redirect
https://passnavi.evidus.com/ct.html?ad=tokai&page=special_plan&number=1&uri=/sp/tokai
`@@||google-analytics.com/analytics.js$domain=passnavi.evidus.com`

Reply:
Looks fine without logged in, I'll test with logged in also with that rule added. For the redirect
`||google-analytics.com/analytics.js$script,redirect=google-analytics,domain=passnavi.evidus.com`
works for supported platforms.

Reply:
Ok, added in my commit.